server/controllers/getTypes.js

```diff
@@ -0,0 +1,25 @@
+const Airtable = require('airtable');
+
+const { Airtable_API_KEY } = process.env;
+if (!process.env.Airtable_API_KEY) {
+  throw new Error('Missing Airtable API KEY env var');
+}
+
+const base = new Airtable({ apiKey: 'keyxrF4iyiMUf7pCB' }).base('appAZnpLnWP0wjAc6');
```

Comment:
why u shared our api key "ya lahwe" :man_facepalming:

Reply:
😱😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱 😱

Reply:
@Hadeel-Salamin sharing is caring ;)

Reply:
@orjwan98 hate masareke :p